data/net/minecraft/world/level/levelgen/carver/WorldCarver.mapping

```diff
@@ -1,34 +1,64 @@
 CLASS net/minecraft/world/level/levelgen/carver/WorldCarver
 	METHOD <init> (Lcom/mojang/serialization/Codec;)V
 		ARG 1 codec
+	METHOD canReach (Lnet/minecraft/world/level/ChunkPos;DDIIF)Z
+		ARG 0 chunkPos
+		ARG 1 x
+		ARG 3 z
+		ARG 5 branchIndex
+		ARG 6 branchCount
+		ARG 7 width
 	METHOD canReplaceBlock (Lnet/minecraft/world/level/block/state/BlockState;)Z
 		ARG 1 state
 	METHOD canReplaceBlock (Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/block/state/BlockState;)Z
 		ARG 1 state
 		ARG 2 aboveState
 	METHOD carve (Lnet/minecraft/world/level/levelgen/carver/CarvingContext;Lnet/minecraft/world/level/levelgen/carver/CarverConfiguration;Lnet/minecraft/world/level/chunk/ChunkAccess;Ljava/util/function/Function;Ljava/util/Random;Lnet/minecraft/world/level/levelgen/Aquifer;Lnet/minecraft/world/level/ChunkPos;Ljava/util/BitSet;)Z
+		COMMENT Carves the given chunk with caves that originate from the given {@code chunkPos}.
+		COMMENT This method is invoked 289 times in order to generate each chunk (once for every position in an 8x8 chunk radius centered around the target chunk).
+		COMMENT
+		COMMENT @see net.minecraft.world.level.chunk.ChunkGenerator#applyCarvers
 		ARG 1 context
 		ARG 2 config
 		ARG 3 chunk
+			COMMENT The chunk to be carved
+		ARG 4 biomeAccessor
 		ARG 5 random
 		ARG 6 aquifer
 		ARG 7 chunkPos
+			COMMENT The chunk position this carver is being called from
 		ARG 8 carvingMask
 	METHOD carveBlock (Lnet/minecraft/world/level/levelgen/carver/CarvingContext;Lnet/minecraft/world/level/levelgen/carver/CarverConfiguration;Lnet/minecraft/world/level/chunk/ChunkAccess;Ljava/util/function/Function;Ljava/util/BitSet;Ljava/util/Random;Lnet/minecraft/core/BlockPos$MutableBlockPos;Lnet/minecraft/core/BlockPos$MutableBlockPos;Lnet/minecraft/world/level/levelgen/Aquifer;Lorg/apache/commons/lang3/mutable/MutableBoolean;)Z
+		COMMENT Carves a single block, replacing it with the appropiate state if possible, and handles replacing exposed dirt with grass.
 		ARG 1 context
 		ARG 2 config
 		ARG 3 chunk
+		ARG 4 biomeAccessor
 		ARG 5 carvingMask
 		ARG 6 random
+		ARG 7 pos
+			COMMENT The position to carve at. The method does not mutate this position.
+		ARG 8 mutablePos
```

Comment:
I would name this "checkPos" since it's used to check for grass and dirt. It's not the best name but it's better than nothing haha